README.md

```diff
@@ -158,6 +158,42 @@ Check if a path starts with `./` or `../`.
 isRelative('./foo')
 ```
 
+### `withHttp`
+
+Ensures url protocol is `http`
+
+```ts
+// Result: http://example.com
+withHttp('https://example.com')
+```
+
+### `withHttps`
+
+Ensures url protocol is `https`
+
+```ts
+// Result: https://example.com
+withHttps('http://example.com')
+```
+
+### `withProtocol`
+
+Changes url protocol passed as second argument
+
+```ts
+// Result: ftp://example.com
+withProtocol('http://example.com', 'ftp:')
+```
+
+### `withoutProtocol`
+
+Removes url protocol
+
+```ts
+// Result: //example.com
```

Comment:
Honestly, I don't like this behavior but we can't be sure always to remove `//` or not remove it...

Reply:
I guess we should remove it and return `example.com`. `//` is not a really valid scheme and even for hasProtocol is only allowed behaind a flag.

Reply:
Done, I've added additional tests as well. Let me know if there is anything left.

Happy to help!